extensions/amp-timeago/1.0/component.js

```diff
@@ -60,6 +60,11 @@ export function Timeago({
       return undefined;
     }
     const observer = new win.IntersectionObserver((entries) => {
+      let {lang} = node.ownerDocument.documentElement;
+      if (lang === 'unknown') {
```

Comment:
Never seen "unknown", but I do see an empty string. There's also `navigator.language` that could be helpful for defaults.

Reply:
I got `unknown` from this reference: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/lang, should I remove this check?

re: package.json changes, this was done in #32809 for 0.1 separately.

Reply:
Going to merge this PR for now, let's follow up separately if we decide to remove `"unknown"`